src/nih_wayfinding/app/scripts/geocoder/search-service.js

```diff
@@ -16,8 +16,23 @@
         ].join(',');
 
         // Public Interface
+        // TODO: Expose the bbox, outFields and categories vars as config options
```

Comment:
Even if it's not exposed as a config parameter yet, could we at least DRY out the bbox from the search and suggest URLs for now?